Lib/unittest/test/testmock/test_sealable.py

```diff
@@ -0,0 +1,185 @@
+import unittest
```

Comment:
Please rename file name to `testsealable.py` to follow the mock module standard.

Reply:
done